creusot-contracts/src/ghost/ptr_own.rs

```diff
@@ -229,6 +232,7 @@ impl<T: ?Sized> PtrOwn<T> {
     ///
     /// Creusot will check that all calls to this function are indeed safe: see the
     /// [type documentation](PtrOwn).
+    #[cfg(feature = "std")] // TODO: Remove std dependency ?
```

Comment:
This function needs to deallocate memory, so it clearly depends on std. Why do you want to break this dep?